sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/options/package-info.java

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * Package containing the classes which serve as options to APIs.
+ */
+package com.azure.search.documents.options;
```

Comment:
We have several other options in indexes.
Do we only put seachClient related options in option bag?

Reply:
Moving classes back to original location as they don't fit the guideline for option bags.